doc/user_guide.md

```diff
@@ -0,0 +1,117 @@
+statements and is intended to replace ubiquitous string concatenation solutions
+which make the code hard to read and are prone to errors and security risks.
+
+SQL Statement Builder allows users to create a SQL statement classes using
+[fluent programming][fluent] that can render into a string.
+
+**Currently supported SQL statements:**
+
+- [CREATE TABLE](#create-table)
+- SELECT
+- INSERT INTO
+
+## CREATE TABLE
+
+The `CreateTable` class of the Sql Statement Builder implements SQL CREATE TABLE
+functionality.
+
+### Usage
+
+1. Create an instance of the `CreateTable` class through the `StatementFactory`
+
+  ```java
+  CreateTable createTable = StatementFactory.getInstance().createTable("tableName");
+  ```
+
+2. Create columns with desired data types using fluent programming
+
+  The column order in a final rendered string will be the same as a sequence of
+  the method calls.
+
+  For example, create a table with three columns (decimal, char and boolean):
+
+  ```java
+  createTable.decimalColumn("col_decimal", 9, 0)
+             .charColumn("col_char", 10)
+             .booleanColumn("col_boolean");
+  ```
+
+  Please keep in mind that the column name if required when creating a column.
+  Additionally, some column types require extra parameters, for instance
+  `VARCHAR`.
+
+  **Currently the following column types are supported:**
+
+  | Column Type                      | Parameter Required | Example                                                          |
+  |:---------------------------------|:-------------------|:-----------------------------------------------------------------|
+  |``BOOLEAN``                       | ``false``          | `createTable.booleanColumn("col_bool")`                          |
+  |``CHAR``                          | ``true``           | `createTable.charColumn("col_char")`                             |
+  |``DATE``                          | ``false``          | `createTable.dateColumn("col_date")`                             |
+  |``DECIMAL``                       | ``true``           | `createTable.decimalColumn("col_dec", 18, 0)`                    |
+  |``DOUBLE PRECISION``              | ``false``          | `createTable.doublePrecisionColumn("col_double_precision")`      |
+  |``INTERVAL DAY TO SECOND``        | ``true``           | `createTable.intervalDayToSecondColumn("col_intdaytosec", 2, 3)` |
+  |``INTERVAL YEAR TO MONTH``        | ``true``           | `createTable.intervalYearToMonthColumn("col_intyeartomonth", 2)` |
+  |``TIMESTAMP``                     | ``false``          | `createTable.timestampColumn("col_timestamp")`                   |
+  |``TIMESTAMP WITH LOCAL TIME ZONE``| ``false``          | `createTable.timestampWithLocalTimeZoneColumn("col_tswithzone")` |
+  |``VARCHAR``                       | ``true``           | `createTable.varcharColumn("col_varchar", 100)`                  |
+
+  You can find more information about the column types in JavaDoc files.
```

Comment:
"... in the SQL Statement Builder's JavaDoc API description."

Reply:
fixed